### DISCUSSION TITLE
Audio message type example

### DISCUSSION THREAD

**Discussion by zhangzhijiang** [flyerhq/flutter_chat_ui#572] [Q&A] [answered]
Does anyone have an chat with audio message example? like enable mic to send an audio message, and receive an audio message ? Thanks

**Comment by zhangzhijiang** [comment]
seems duplicated

**Comment by demchenkoalex** [comment, marked as answer]
pointing to this https://github.com/flyerhq/flutter_chat_ui/discussions/149
